content/trailmap/prologue.md

```diff
@@ -0,0 +1,159 @@
++++
+title = "Dogecoin Trailmap: Prologue"
+date = "2021-12-10"
+type = "about"
+[ author ]
+name = "Dogecoin Foundation"
++++
+
+<section class="presentation">
+<div class="left">
+
+<div class="title">
+
+
+ ## <img title='You are here!' width="60px" style='display: inline;' src="/marker.png"/>Dogecoin Trailmap: Prologue
```

Comment:
need a space at the end of the sentence 'You are here!', otherwise the previous sentence is too close to 'Dogecoin Trailmap'

Reply:
A title tag shouldn't be visible unless you hover the image? How do you see the text running together?